static/js/src/tabbedContent.js

```diff
@@ -83,3 +83,29 @@
     });
   })();
 })();
+
+(function () {
+  // Toggles show board based on selection on small screens
+
+  const boards = document.querySelectorAll(`[role=tabpanel]`);
+  const dropdownSelect = document.getElementById('boardSelect');
+
+  if (dropdownSelect) {
+    dropdownSelect.addEventListener('change', () => {
+      selectBoard();
+    });
+  } else {
+    console.error('dropdownSelect element not found');
+  }
```

Comment:
Is this logging necessary? Can you investigate in what circumstances `dropdownSelect` is not defined and cover as much as possible, say for example you may want to add an [`onload`](https://stackoverflow.com/questions/1033398/how-to-execute-a-function-when-page-has-fully-loaded) listener.

This logging sounds like it should be there mainly for debugging purposes.

Reply:
Leaving it blank was causing linting issues so I logged in the error, but yeah probably added onload could be a way.